src/registry/variants.ts

```diff
@@ -75,14 +80,18 @@ const maybeGetProject = (projectDir?: string): SfProjectJson | undefined => {
   }
 };
 
-const loadPreset = (preset: string): MetadataRegistry => {
-  const pathToCheck = path.join(__dirname, 'presets', `${preset}.json`);
+const loadPresetJson = (): void => {
+  presetsJsonMap.set('decomposeCustomLabelsBeta', decomposeCustomLabelsBeta as MetadataRegistry);
+  presetsJsonMap.set('decomposePermissionSetBeta', decomposePermissionSetBeta as MetadataRegistry);
+  presetsJsonMap.set('decomposeSharingRulesBeta', decomposeSharingRulesBeta as MetadataRegistry);
+  presetsJsonMap.set('decomposeWorkflowBeta', decomposeWorkflowBeta as MetadataRegistry);
+};
 
+const loadPreset = (preset: string): MetadataRegistry => {
   try {
-    const rawPreset = fs.readFileSync(pathToCheck, 'utf-8');
-    return JSON.parse(rawPreset) as MetadataRegistry;
+    return presetsJsonMap.get(preset) as MetadataRegistry;
```

Comment:
fyi, `Map.get` returns `undefined` when you ask for something not in the map.  It doesn't throw an error

Reply:
TS would have told you had a bug, but using ` as` hides the error from yourself